src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -259,4 +263,31 @@ default GameModeData getGameModeData() {
      */
     void setSleepingIgnored(boolean sleepingIgnored);
 
+    /**
+     * Gets a collection of {@link ServerBossBar}s this player is a member of.
+     *
+     * <p>The returned collection is unmodifiable.</p>
+     *
+     * @return The boss bars
+     */
+    Collection<ServerBossBar> getBossBars();
```

Comment:
IMO we should keep it a `Collection` to avoid any unnecessary requirements for impls.
Similar to what we have done with [`GameRegistry#getAllOf(Class<T>)`](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/GameRegistry.java#L112).